docs/enterprise/helm-guide.md

```diff
@@ -0,0 +1,228 @@
+   apiVersion: v1
+   kind: Secret
+   type: Opaque
+   metadata:
+     name: feldera-db-secret
+   stringData:
+     .user: "..."  # Fill in
+     .password: "..."  # Fill in
+   ```
+
+2. **Create secret:** with the configuration file, create the secret:
+   ```bash
+   #kubectl create namespace feldera  # If the namespace does not exist yet
+   kubectl apply -n feldera -f feldera-db-secret.yaml
+   ```
+
+3. **Specify secret during installation:** in the `helm` installation command, set
+   the following:
+   ```
+   --set felderaDatabaseSecretRef="feldera-db-secret"
+   ```
+   _Note:_ it must be a new installation.
+
+### Feldera installation overview
+
+**Services:**
+```
+$ kubectl get svc -n feldera
+NAME                        TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)     AGE
+feldera-prober-server       ClusterIP   10.43.67.233    <none>        44444/TCP   5m10s
+feldera-kubernetes-runner   ClusterIP   10.43.175.228   <none>        8080/TCP    5m10s
+feldera-api-server          ClusterIP   10.43.176.120   <none>        8080/TCP    5m10s
+feldera-db                  ClusterIP   10.43.60.190    <none>        5431/TCP    5m10s
+feldera-compiler-server     ClusterIP   10.43.60.125    <none>        8085/TCP    5m10s
+```
+
+**Pods:**
+```
+$ kubectl get pods -n feldera
+NAME                                         READY   STATUS    RESTARTS   AGE
+feldera-db-0                                 1/1     Running   0          5m31s
+feldera-kubernetes-runner-6447b8f56d-86j4w   1/1     Running   0          5m31s
+feldera-prober-server-5c945b7fcf-9r727       1/1     Running   0          5m31s
+feldera-compiler-server-0                    1/1     Running   0          5m31s
+feldera-api-server-c546499bc-wdpkm           1/1     Running   0          5m31s
+```
+
+**StatefulSets:**
+```
+$ kubectl get sts -n feldera
+NAME                      READY   AGE
+feldera-db                1/1     5m41s
+feldera-compiler-server   1/1     5m41s
+```
+
+**Deployments:**
+```
+$ kubectl get deployments -n feldera
+NAME                        READY   UP-TO-DATE   AVAILABLE   AGE
+feldera-kubernetes-runner   1/1     1            1           5m55s
+feldera-prober-server       1/1     1            1           5m55s
+feldera-api-server          1/1     1            1           5m55s
+```
+
+**Persistent volume claims:**
+```
+$ kubectl get pvc -n feldera
+NAME                                         STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS   AGE
+db-storage-feldera-db-0                      Bound    pvc-a5acb4ef-1d3c-4773-ad42-528657f11c94   20Gi       RWO            local-path     6m11s
+compiler-storage-feldera-compiler-server-0   Bound    pvc-07369993-c20a-4f8a-abd1-52bf3be413c3   20Gi       RWO            local-path     6m11s
+```
+
+### Uninstallation
+
+The most direct and thorough way to uninstall Feldera can be done
+if it was installed in its own dedicate namespace by deleting the
```

Comment:
If you installed Feldera in its own dedicated namespace, the most thorough way to uninstall Feldera is to delete the entire namespace.